packages/@okta/vuepress-site/docs/reference/user-query/index.md

```diff
@@ -0,0 +1,716 @@
+        },
+    },
+    {
+        "id": "00u269cmneNMFHCH51d7",
+        "status": "PROVISIONED",
+        "created": "2021-12-17T15:16:55.000Z",
+        "activated": null,
+        "statusChanged": null,
+        "lastLogin": null,
+        "lastUpdated": "2022-05-24T15:38:04.000Z",
+        "passwordChanged": "2021-12-17T15:16:56.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "Janice",
+            "lastName": "Benson",
+            "mobilePhone": null,
+            "secondEmail": null,
+            "department": "Engineering",
+            "login": "janice.benson@gmail.com",
+            "email": "janice.benson@gmail.com"
+        },
+     }
+```
+
+### Limits and pagination
+
+You can include the parameters `limit` and `after` in search expressions to limit the return of user records and to access the [pagination](/docs/reference/core-okta-api/#pagination) cursor location. By using the `after` parameter with the `limit` parameter, you can define the cursor location in the data set and manage the user records per page. The cursor is an opaque value that is obtained through the link header [next link relation](/docs/reference/core-okta-api/#link-header).
+
+> **Note:** If you don't specify a value for limit, the maximum (200) is used as a default.
+
+#### Request example
+
+The following example returns users who have an `ACTIVE` status and limits the records returned to five.
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users?search=status+eq+%22ACTIVE%22&limit=5"
+```
+
+#### Response header
+The request returned the first five users and included the following next link relation in the header:
+
+```JSON
+HTTP/1.1 200 OK
+Content-Type: application/json
+Link: <https://{yourOktaDomain}/api/v1/users?search=status+eq+%22ACTIVE%22&limit=5>; rel="self"
+Link:<https://{yourOktaDomain}/api/v1/users?after=00u3p62n11UKppeh31d7&limit=5&search=status+eq+%22ACTIVE%22>; rel="next"
+```
+
+#### Request example
+
+The following example returns the next five users with an `ACTIVE` status using the next link relation:
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users?after=00u3p62n11UKppeh31d7&limit=5&search=status+eq+%22ACTIVE%22"
+```
+
+### Sorting parameters
+
+You can include the sorting parameters `sortBy` and `sortOrder` in search expressions to deliver results sorted by any single user property, for example, returning users sorted by department or employee number. All users returned by a `sortBy` expression are, by default, ordered by their user `id`. Use `sortBy` to specify the property that you want to sort by, and `sortOrder` to specify the order (`asc` and `desc`). By default, the sort order is ascending.
```

Comment:
Should (`asc` and `desc`) instead be (`asc` or `desc`)?  It's not like a request can specify both!

Reply:
Very true. Updated. Thanks.